src/OrleansRuntime/GrainDirectory/MultiClusterRegistration/GlobalSingleInstanceRegistrar.cs

```diff
@@ -33,14 +34,14 @@ internal class GlobalSingleInstanceRegistrar : IGrainRegistrar<GlobalSingleInsta
 
         public GlobalSingleInstanceRegistrar(
             LocalGrainDirectory localDirectory,
-            Factory<string, Logger> loggerFactory,
+            ILoggerFactory loggerFactory,
```

Comment:
nit: can just inject the generic version, as the factory is never used after creating the logger